CHANGELOG.md

```diff
@@ -2,83 +2,88 @@
 
 ## Unreleased
 
+## 1.0.0
+
+- Resolve all link fields found within "items" in an API response, and replace the links with the actual entities if available from the "includes" section of API response. Inspired by https://github.com/contentful/contentful.js/blob/master/ADVANCED.md#link-resolution
+- Breaking change: 'assets' field on Entry struct is no longer populated. All Essets for an Entry are embedded directly within 'fields' now instead
+
 ## 0.5.0
 
-* [#97](https://github.com/contentful-userland/contentful.ex/pull/97) switches underlying http adapter to `tesla` for higher flexibility. Thank you @OldhamMade
-* tests against Elixir 1.13 and updates required elixir version to 1.11
+- [#97](https://github.com/contentful-userland/contentful.ex/pull/97) switches underlying http adapter to `tesla` for higher flexibility. Thank you @OldhamMade
```

Comment:
my markdown formatter removed these, I don't think it needs both a `-` and a `*` to format correctly